.github/workflows/ci.yaml

```diff
@@ -75,7 +78,7 @@ jobs:
 
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '2.7.2'
+          ruby-version: "2.7.2"
```

Comment:
Might be worth bumping to `3.0.0` or `3.1.0` as 3.0 release drastically improved performance.

Reply:
If cocoapod (and anyone else in the ci pipeline) supports, I have no problem :)